boardswarm/src/registry.rs

```diff
@@ -36,6 +36,9 @@ impl Properties {
         self.properties.get(prop).map(String::as_ref)
     }
 
+    /// Tests if the elements match all of the elements of an iterator.
+    ///
+    /// An empty iterator returns `true`.
```

Comment:
```suggestion
    /// Tests if matches is a subset of properties
    ///
    /// If properties is from a remote instance (`boardswarm.instance` is set)  that has to be 
    /// explicitly matched otherwise it's a pure subset match (e.g. an empty set matches)
```

We're not passed an iterator we're passed something that can be turned into an iterator ; also i don't think i've seen

Reply:
ok